Dungeon_of_Munchkinomicon.py

```diff
@@ -20,19 +20,28 @@ def death(name, reason):
 	print "You died because %s. What did you think would happen?" % reason
 	redo(name)
 	
+def getConfirmation(ask):

```

Comment:
There were a couple places where you were looking for a yes or a no from the user so I moved that into its own function.